File: wrappers/dotnet/indy-sdk-dotnet/WalletApi/WalletItemAlreadyExistsException.cs
namespace Hyperledger.Indy.WalletApi
{
    /// <summary>
    /// Exception thrown when value with the specified key already exists in the wallet.
    /// </summary>
    /// <seealso cref="Hyperledger.Indy.IndyException" />
    public class WalletItemAlreadyExistsException : IndyException
    {
        private const string message =
            "The specified item already exists in the wallet.";

        internal WalletItemAlreadyExistsException() : base(message, (int)ErrorCode.WalletItemAlreadyExistsError)
        {
        }
    }
}

File: wrappers/java/src/main/java/org/hyperledger/indy/sdk/wallet/WalletItemAlreadyExistsException.java
package org.hyperledger.indy.sdk.wallet;

import org.hyperledger.indy.sdk.ErrorCode;
import org.hyperledger.indy.sdk.IndyException;

/**
 * Exception thrown when add record operation is used with record name that already exists.
 */
public class WalletItemAlreadyExistsException extends IndyException
{
	private static final long serialVersionUID = 667964860056778208L;
	private final static String message = "Item already exists.";

	/**
	 * Initializes a new WalletItemNotFoundException.
	 */
	public WalletItemAlreadyExistsException() {
		super(message, ErrorCode.WalletItemAlreadyExists.value());
	}
}
